composer.json

```diff
@@ -16,6 +16,7 @@
     "laminas/laminas-cache-storage-implementation": "1.0"
   },
   "require-dev": {
+    "ext-apc": "*",
```

Comment:
I've seen this only now: may I ask you why it's in the `require-dev` section instead of the `require` one?

Reply:
Adding it to `require` would introduce a BC break. Extensions are being required in the next major when bumping to `laminas-cache` v3.

Reply:
> Adding it to `require` would introduce a BC break.

It doesn't, is it @Ocramius ?
I've always considered it like bumping PHP version: ok in a MINOR :monocle_face:

Reply:
Hm, maybe in a minor this might be okay. In the initial version we avoided this due to the fact that users would be prevented from upgrading to `laminas-cache` v2.10. 
But true, we could add this in a new minor version here.
This thing became way complicated 😅